packages/ra-core/src/form/getSimpleValidationResolver.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { getSimpleValidationResolver } from './getSimpleValidationResolver';
+
+describe('getSimpleValidationResolver', () => {
+    const validator = getSimpleValidationResolver(errors => errors);
```

Comment:
this is misleading: the parameter passed to the resolver callback parameter should be named values, not errors